scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -203,6 +207,25 @@ private static BuildTargetResult CompileStage(BuildTargetContext c, DotNetCli do
             File.Delete(Path.Combine(binDir, $"dotnet{Constants.ExeSuffix}"));
             File.Copy(Path.Combine(binDir, $"corehost{Constants.ExeSuffix}"), Path.Combine(binDir, $"dotnet{Constants.ExeSuffix}"));
 
+            // HACK
+            // bootstrapping for Windows x86. Copy csc/vbc from stage0.
+            // This is a temporary hack for https://github.com/dotnet/roslyn/issues/8951
+            if (IsWinx86)
```

Comment:
I am not sure if at this point the x86 logic is too different to warrant its own BuildTarget.